packaging/functions.sh

```diff
@@ -244,6 +244,7 @@ set_mod_version() {
 # Copy launch wrappers, application icons, desktop, and MIME files to the target directory
 # Arguments:
 #   SRC_PATH: Path to the root OpenRA directory
+#   DEST_PATH: Prefix for the install path (DON'T put it into anything as it will break Linux packaging)
```

Comment:
I don't quite understand the comment here. If nothing should be put in `DEST_PATH` why would we need the variable in the first place.

Reply:
I rephrased it. https://en.opensuse.org/openSUSE:Packaging_checks#file-contains-buildroot doesn't really explain it either. If you install (temporarily) into `home/user/rpm/usr/bin/openra-d2k` and that script launches in `home/user/rpm/usr/lib/openra` and then move the folder, it should still work. If you hard-coded that absolute `home/user/rpm/usr/lib/openra` folder everything will break. If you separate `home/user/rpm` (called `DESTDIR` in `Makefile` or `%{buildroot}` in .rpm specs from the FHS folder structure it is supposed to go, that problem vanishes. This is why the current playtest in its current form can't be repackaged. Debian/Ubunut will have the same problem (or apply patches) which is always annoying, but common case when packaging games which are usually badly maintained upstreams.

Reply:
Does https://www.debian.org/doc/manuals/maint-guide/modify.en.html#destdir suggest that this should instead be set in the Makefile (is https://www.gnu.org/prep/standards/html_node/DESTDIR.html#DESTDIR any help)?

Reply:
That is nothing new and if you look at my https://build.opensuse.org/package/view_file/games:openra/playtest/openra.spec you will see how it uses

```spec
make DESTDIR=%{buildroot} prefix=%{_prefix} install
```

The OpenRA way is a bit intricate as it doesn't use pure `Makefile` syntax for everything, but instead outsources the details of creating wrapper scripts and installing them into separate bash scripts. That is indeed unique and may be confusing at first.

Reply:
And one last question if you don't mind: why not naming it DESTDIR in the scripts too so the packing people know "ah right, the DESTDIR, I can't use this for anything but during install". Or is `DEST_PATH` likewise common sense?